### DISCUSSION TITLE
Welcome to R-Instat Discussions!

### DISCUSSION THREAD

**Discussion by dannyparsons** [IDEMSInternational/R-Instat#6236] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community or start a new discussion topic if you have something specific to share.

**Comment by Sabisika5** [comment]
When we try to recode the factors, R-instat (0.7.16.50 version, dialog: prepare, column factor, recode factor) stops working and the page closes. How to correct this?
